static/templates/settings/account_settings.hbs

```diff
@@ -180,7 +180,7 @@
                 <button class="button" data-dismiss="modal">{{t "Cancel" }}</button>
                 <button class="button btn-danger rounded" id="do_deactivate_self_button">
                     <object class="loader" type="image/svg+xml" data="/static/images/loader.svg"></object>
-                    <span>{{t "Deactivate now" }}</span>
+                    <span>{{t "Confirm" }}</span>
```

Comment:
<img width="427" alt="deactivate_user_modal" src="https://user-images.githubusercontent.com/59444243/113509281-5d5b3180-9572-11eb-8b45-2ef1afb7529b.PNG">

Reply:
Yes, let's add: "Are you sure you want to deactivate your account?"